doc/source/user/absolute_beginners.rst

```diff
@@ -0,0 +1,2451 @@
+Installing NumPy
+----------------
+  
+To install NumPy, we strongly recommend using a scientific Python distribution. If you're looking for the full instructions for installing NumPy on your operating system, you can `find all of the details here <https://www.scipy.org/install.html>`_.
+
+
+  
+If you already have Python, you can install NumPy with
+
+::
+
+  conda install numpy
+  
+or 
+
+::
+
+  pip install numpy
+  
+If you don't have Python yet, you might want to consider using `Anaconda <https://www.anaconda.com/>`_. It's the easiest way to get started. The good thing about getting this distribution is the fact that you don’t need to worry too much about separately installing NumPy or any of the major packages that you’ll be using for your data analyses, like pandas, Scikit-Learn, etc.
+
+You can find all of the installation details in the `Installation <https://www.scipy.org/install.html>`_ section at scipy.org.
+
+How to import NumPy
+-------------------
+
+Any time you want to use a package or library in your code, you first need to make it accessible. 
+
+In order to start using NumPy and all of the functions available in NumPy, you'll need to import it. This can be easily done with this import statement:
+
+::
+
+  import numpy as np 
+
+(We shorten "numpy" to "np" in order to save time and also to keep code standardized so that anyone working with your code can easily understand and run it.)
+
+What’s the difference between a Python list and a NumPy array? 
+--------------------------------------------------------------
+  
+NumPy gives you an enormous range of fast and efficient numerically-related options. While a Python list can contain different data types within a single list, all of the elements in a NumPy array should be homogenous. The mathematical operations that are meant to be performed on arrays wouldn't be possible if the arrays weren't homogenous. 
+
+**Why use NumPy?**
+
+NumPy arrays are faster and more compact than Python lists. An array consumes less memory and is convenient to use. NumPy uses much less memory to store data and it provides a mechanism of specifying the data types. This allows the code to be optimized even further. 
+
+What is an array?
+-----------------
+
+An array is a central data structure of the NumPy library. An array is a grid of values and it contains information about the raw data, how to locate an element, and how to interpret an element. It has a grid of elements that can be indexed in `various ways <https://numpy.org/devdocs/user/quickstart.html#indexing-slicing-and-iterating>`_. The elements are all of the same type, referred to as the array ``dtype``. 
+
+An array can be indexed by a tuple of nonnegative integers, by booleans, by another array, or by integers. The ``rank`` of the array is the number of dimensions. The ``shape`` of the array is a tuple of integers giving the size of the array along each dimension.
+
+One way we can initialize NumPy arrays is from nested Python lists. 
```

Comment:
How about "... from Python lists, using nested lists for two- or higher-dimensional data."?

Reply:
Perhaps good to show a 1-D example first?

Reply:
Got it!